modules/hardpoints/ax_missile_rack.json

```diff
@@ -3,6 +3,7 @@
     {
       "experimental": true,
       "ammo": 64,
+      "ammocost": 235,
```

Comment:
Is this for a full reload or 1 clip?

Reply:
It is a single shot. This was already present for AFNMU, Chaff and Heatsinks.

Reply:
awesome thanks for clearing that up